src/ui/common/src/components/Search/index.tsx

```diff
@@ -86,7 +82,7 @@ export const filteredList = (
   matchOn: (item: any) => string,
   listItems: (item: any, idx: number) => JSX.Element,
   noItemsMessage: JSX.Element
-) => {
+): JSX.Element => {
```

Comment:
Out of curiosity what is `JSX.Element` vs `React.Element`?

Reply:
It's a more generic version of React.Element.

More info, and probably more than you'd like to know about these two here:
https://stackoverflow.com/questions/58123398/when-to-use-jsx-element-vs-reactnode-vs-reactelement

As for which one to use? I'm not 100% sure, they both seem to do the job and I don't really have strong opinions here.